src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -510,8 +524,28 @@ export default {
 		this.partnerName = loan?.partnerName ?? '';
 		this.partnerCountry = loan?.partner?.countries[0]?.name ?? '';
 		this.lender = data?.my?.userAccount ?? {};
+
+		// ACK-469 Experiment Tracking for generating the scle in utm_campaign
+		const shareLanguageBpExp = getExperimentSettingCached(this.apollo, 'share_language_bp');
+		// Tracking for EXP-ACK-469-Apr2023
+		if (shareLanguageBpExp?.enabled) {
```

Comment:
We don't need to check `enabled` anymore and won't need to add that to experiment settings moving forward. The `trackExperimentVersion` already checks that the experiment version is defined before logging:

```
if (exp.version && exp.version !== 'unassigned') {
```

Reply:
Oh yup, the experiment assignment that we add to the component prefetch will get the experiment settings either from cache or async, if needed. And the assignment checks the `active_experiments` before assigning (replacing the enabled check).